src/SlackConnector/Connections/Clients/Chat/FlurlChatClient.cs

```diff
@@ -36,6 +37,29 @@ public FlurlChatClient(IResponseVerifier responseVerifier)
 
             var response = await request.GetJsonAsync<StandardResponse>();
             _responseVerifier.VerifyResponse(response);
+            return response.Ts;
```

Comment:
Instead of returning a string, can you return a model? It currently isn't very intuitive